README.md

```diff
@@ -92,7 +92,7 @@ delete debugMenu.customEntries['myKey']
 
 ### Some debugging examples
```

Comment:
Not something you necessarily need to change; but as a note to myself and other maintainers, it's probably worth deciding if we want to capitalise section titles or not.